src/arrayMethodSort.js

```diff
@@ -3,10 +3,29 @@
 /**
  * Implement method Sort
  */
-function applyCustomSort() {
+function applyCustomSort(callback) {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
just make a default function !compareFunction
```suggestion
  [].__proto__.sort2 = function(compareFunction = someFunction) {
```